libr/config/config.c

```diff
@@ -217,8 +217,6 @@ R_API void r_config_list(RConfig *cfg, const char *str, int rad) {
 			if (!str || (str && (!strncmp (str, node->name, len)))) {
 				if (!str || !strncmp (str, node->name, len)) {
```

Comment:
these conditionals look like very similar ones to the ones in the line above, this can be probably improved with caching the results in variables instead

Reply:
OK. Let me try a few things for the conditions and the single/double question mark parsing, because e??search also did not show the variable names. I'll update the pull request later